src/backer/queueing.py

```diff
@@ -0,0 +1,44 @@
+from hio import doing
+from keri.db import subing
+from keri.core import serdering, eventing
+from ... import cardaning
+
+
+class Queueing(doing.Doer):
+
+    def __init__(self, name='backer', hab=None):
+        self.name = name
+        self.hab = hab
+        # TODO: pending_kel should change to array
+        self.pending_kel = None
+        self.ledger = cardaning.Cardano(name=name, hab=hab)
+        # sub-dbs
+        self.keldb_queued = subing.SerderSuber(db=hab.db, subkey="kel_queued")
+        self.keldb_published = subing.SerderSuber(db=hab.db, subkey="kel_published")
+        self.tock = 10  # the job should run daily.
+
+    def publish(self):
+        """
+        doer call run
+            get from queued
+                do publishEvent
+            deleted from queued
+            create in published
+        """
+        for (pre, said), serder in self.keldb_queued.getItemIter():
+            event = eventing.loadEvent(self.hab.db, pre, serder.saidb)
```

Comment:
is it possible that this `loadEvent` can fail if it didn't parse properly, what happens then? Can you test this and see how we can handle it

Reply:
add the exception
pls ref the latest commit: a54d02f52ce5fd090055598ddc584c4a25768e54

Reply:
hmm, but what I mean is this: what happens if you send a "bad" event to the API (maybe you can try test this)

Do we still call `self.queue.push_to_queued` or does it exit earlier?

Reply:
ah, got your point now, will add test case that send to blockfrost

Reply:
hmm when I say API, I mean the cardano-backer endpoint, not blockfrost

`HttpEnd` onPost

Reply:
updated
ref here: 992d92335d82486cb7ce0ea0129ab0a6619be2ed